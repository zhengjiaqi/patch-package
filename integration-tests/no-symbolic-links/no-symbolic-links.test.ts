import { runIntegrationTest } from "../runIntegrationTest"
runIntegrationTest({
  projectName: "no-symbolic-links",
  shouldProduceSnapshots: true,
})
