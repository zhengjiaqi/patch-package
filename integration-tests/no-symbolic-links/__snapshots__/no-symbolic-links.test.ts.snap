// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test no-symbolic-links: patch-package fails to create a patch when there are symbolic links 1`] = `
"SNAPSHOT: patch-package fails to create a patch when there are symbolic links

⛔️ ERROR

  Your changes involve creating symlinks. patch-package does not yet support
  symlinks.
  
  ️Please use --include and/or --exclude to narrow the scope of your patch if
  this was unintentional.

END SNAPSHOT"
`;
